### PR TITLE
Fix k8s-manifests for ci

### PR DESCRIPTION
**Pull request by shapovalovdev**

### CODE REVIEW

File: common/common.sh [hunk 1]
```diff
@@ -2,6 +2,17 @@
 
 set -o errexit
 
+#install net-tools
+
+source /etc/os-release
```
Comment (alexey-mr) [inline common/common.sh:7]: Look below, there is already read of that file.

File: common/common.sh [hunk 2]
```diff
@@ -2,6 +2,17 @@
...
+ID=${ID:-centos}
+LINUX_DISTRIB=$ID
+if [[ ${LINUX_DISTRIB} == 'centos' ]]; then
```
Comment (alexey-mr) [inline common/common.sh:10]: see startup.sh#L23 
there are sections for installation of requried packages , just add the package there.

**Review by alexey-mr — COMMENTED** [review]
see comments

**Comment by vmbrasseur** [comment]
Hey there, gang. We don't accept PRs to this repo. tf-devstack development happens over on Gerrit: https://gerrit.tungsten.io/r/admin/repos/tungstenfabric/tf-devstack

Please close this PR and move the patch over to Gerrit. Thanks!

**Comment by shapovalovdev** [comment]
pull requests are no allowed.

**Pull request closed without merging**